src/plugins/knex.js

```diff
@@ -62,11 +62,12 @@ function createWrapRunnerQuery (tracer, config) {
 }
 
 function addError (span, error) {
-  span.setTag('error', 'true')
-  span.log({
-    'error.type': error.name,
-    'error.msg': error.message,
-    'error.stack': error.stack
+  span.addTags({
+    'sfx.error': true,
+    'sfx.error.kind': error.name,
+    'sfx.error.object': error.name,
+    'sfx.error.message': error.message,
+    'sfx.error.stack': error.stack
```

Comment:
Turns out knex integration actually fails to capture errors right now. Updated code to use SignalFx error tags for now. Will address the actual error handling issue in an upcoming PR.